LIZARDS SHALL FREEZE!!!

loltart opened the pull request:
<!--
SPDX-FileCopyrightText: 2021 Pieter-Jan Briers <pieterjan.briers+git@gmail.com>
SPDX-FileCopyrightText: 2021 Swept <sweptwastaken@protonmail.com>
SPDX-FileCopyrightText: 2021 mirrorcult <lunarautomaton6@gmail.com>
SPDX-FileCopyrightText: 2022 AJCM-git <60196617+AJCM-git@users.noreply.github.com>
SPDX-FileCopyrightText: 2022 Kara <lunarautomaton6@gmail.com>
SPDX-FileCopyrightText: 2023 DrSmugleaf <DrSmugleaf@users.noreply.github.com>
SPDX-FileCopyrightText: 2023 Kevin Zheng <kevinz5000@gmail.com>
SPDX-FileCopyrightText: 2024 Vasilis <vasilis@pikachu.systems>
SPDX-FileCopyrightText: 2024 lzk <124214523+lzk228@users.noreply.github.com>
SPDX-FileCopyrightText: 2025 Aiden <28298836+Aidenkrz@users.noreply.github.com>

SPDX-License-Identifier: AGPL-3.0-or-later
-->

<!-- Guidelines: https://docs.spacestation14.io/en/getting-started/pr-guideline -->
<!-- NOTE: All code submitted to this repository is ALWAYS licensed under the AGPL-3.0-or-later license. 
The REUSE Specification headers or separate .license files indicate a secondary license (e.g., MPL or MIT) solely to facilitate 
integration for projects that do not use the AGPL license. This secondary license does not replace the fact that AGPL-3.0-or-later remains the primary and binding license. 
Uncomment and modify the following line if you wish to change the license from the default of AGPL.-->
<!--- LICENSE: AGPL -->
## About the PR
<!-- What did you change? -->
90% of this was already done by solstice
Cold blood lizards. They get a heat lamp round start. Multiple tiers of heatlamps in research. Heatoninteract component. Syndicate heatlamp. Lizards can be in hotter environments. Lizards start taking cold damage quicker. They take more cold damage than before.

Reasons it was reverted before:

They didn't spawn with heatlamps a decent portion of the time - fixed
Heatlamps died within twenty seconds - fixed
People were malding - Mald harder
"just make it a trait" - screw you I'll make your tail drag a trait and force you to take this if you want to keep it
Lizards got cold in space even while using a space suit - fixed
 
## Why / Balance
<!-- Discuss how this would affect game balance or explain why it was changed. Link any relevant discussions or issues. -->
They should freeze.
## Technical details
<!-- Summary of code changes for easier review. -->
They deserve a proper downside to their race and having more races with drastic downsides is fun
## Media
<!-- Attach media if the PR makes ingame changes (clothing, items, features, etc).
Small fixes/refactors are exempt. Media may be used in SS14 progress reports with credit. -->


# Showcase video
https://youtu.be/dMSs4ePdSzQ
Everything's been tested and works

## Requirements
<!-- Confirm the following by placing an X in the brackets [X]: -->
- [X] I have read and am following the [Pull Request and Changelog Guidelines](https://docs.spacestation14.com/en/general-development/codebase-info/pull-request-guidelines.html).
- [X] I have added media to this PR or it does not require an ingame showcase.
<!-- You should understand that not following the above may get your PR closed at maintainer’s discretion -->

## Breaking changes
<!-- List any breaking changes, including namespaces, public class/method/field changes, prototype renames; and provide instructions for fixing them.
This will be posted in #codebase-changes. -->

**Changelog**

Loltart, Solstice
:cl:
- add: Lizards are now cold blooded!
- add: New researchable heatlamps. For when the classic one just isn't enough.
- add: Lizards now get a heatlamp when they spawn.
- add: Syndicate heatlamp. Don't let sec steal it!
- tweak: Lizards are more resistance to high heat environments (go play in the plasmafires - just don't get caught on fire yourself!)
- tweak: Lizards get colder faster, and take more damage from cold environments.
- tweak: The chemical Herignis has been reworked! Check it out in the guidebook!
- tweak: You can now craft normal heatlamps in the medfab roundstart.
- fix: Heatlamps keep lizards warm in space while using a space suit

ImWeax commented:
bro this was reverted because nobody liked it and it was buggy as hell - bounty was to make it a trait so make it a trait.

TurboTrackerss14 commented:
>bounty to make trait
>no trait code in sight
>just reused old reverted code with fixes
>31k bounty
🥹 

I really would think you would put deeper effort into this considering the pricetag.

arguement of ss13 coldblooded was larp, and goalposts moved after I corrected you. 
argument of vox negative was larp, because no organ code here. (until changes offered this was recent) 

https://www.youtube.com/watch?v=rQ6y5ZpUcT8

Terraspark4941 commented:
MAINTAINERS DO NOT MERGE THIS SHIT **UNTIL** IT HAS 31K GOOBCOINS WORTH OF EFFORT IN BUGTESTING
I WILL NOT STAND FOR ANOTHER SHITBUILD ON MASTER

PunishedJoe commented:
You've had it too good for too long

Sarahon commented:
This was a bait bounty, somebody really wasted their GC.

TurboTrackerss14 commented:
> This was a bait bounty, somebody really wasted their GC.

Its more that loltart didn't even complete the bounty to begin with 😆

loltart commented:
> bro this was reverted because nobody liked it and it was buggy as hell - bounty was to make it a trait so make it a trait.



> > This was a bait bounty, somebody really wasted their GC.
> 
> Its more that loltart didn't even complete the bounty to begin with 😆

1. it's a race downside that shouldn't be easily ignored by just not picking a trait - they get a free upside of tail grab right now
2. I put effort into this and was going to do various bugfixes
3. I was going to tie this to heat

loltart commented:
> > bounty to make trait
> > no trait code in sight
> > just reused old reverted code with fixes
> > 31k bounty
> > 🥹
> 
> I really would think you would put deeper effort into this considering the pricetag.
> 
> arguement of ss13 coldblooded was larp, and goalposts moved after I corrected you. argument of vox negative was larp, because no organ code here. (until changes offered this was recent)
> 
> https://www.youtube.com/watch?v=rQ6y5ZpUcT8

I said I didn't care at the start yet you continued and then after I pointed it out again you said I moved goalposts. stop being a joke

loltart commented:
author said it had to be a trait which I disagree with so I"m closing this

loltart commented:
> > bro this was reverted because nobody liked it and it was buggy as hell - bounty was to make it a trait so make it a trait.
> 
> > > This was a bait bounty, somebody really wasted their GC.
> > 
> > 
> > Its more that loltart didn't even complete the bounty to begin with 😆
> 
>     1. it's a race downside that shouldn't be easily ignored by just not picking a trait - they get a free upside of tail grab right now
> 
>     2. I put effort into this and was going to do various bugfixes
> 
>     3. I was going to tie this to the heart

ImWeax commented:
vro hell no re-delete this shit rn
